### ISSUE TITLE
[BUG] Nut Extensions throws Exception

### ISSUE THREAD

**Issue by tonicospinelli** [bolt/bolt#3597]
```shell
$ app/nut extensions
PHP Fatal error:  Cannot access private property Bolt\Composer\PackageManager::$messages in /Users/aspinelli/Projects/bolt/src/Nut/Extensions.php on line 28
```

I try to list all installed extensions and I got this exception.

**Comment by GawainLynch** [comment]
What version of Bolt?

**Comment by tonicospinelli** [comment]
on master branch, I will check it in other versions

**Comment by tonicospinelli** [comment]
for branch release/2.0 I got other exception
```shell
PHP Fatal error:  Class 'URLify' not found in /Users/aspinelli/Projects/bolt/src/Helpers/String.php on line 18
PHP Stack trace:
PHP   1. {main}() /Users/aspinelli/Projects/bolt/app/nut:0
PHP   2. require_once() /Users/aspinelli/Projects/bolt/app/nut:6
PHP   3. require_once() /Users/aspinelli/Projects/bolt/app/load.php:40
PHP   4. call_user_func:{/Users/aspinelli/Projects/bolt/app/bootstrap.php:86}() /Users/aspinelli/Projects/bolt/app/bootstrap.php:86
PHP   5. Bolt\{closure}() /Users/aspinelli/Projects/bolt/app/bootstrap.php:86
PHP   6. Bolt\Application->__construct() /Users/aspinelli/Projects/bolt/app/bootstrap.php:79
PHP   7. Bolt\Application->initConfig() /Users/aspinelli/Projects/bolt/src/Application.php:48
PHP   8. Pimple->offsetGet() /Users/aspinelli/Projects/bolt/src/Application.php:73
PHP   9. Bolt\Application::{closure:/Users/aspinelli/Projects/bolt/vendor/pimple/pimple/lib/Pimple.php:122-130}() /Users/aspinelli/Projects/bolt/vendor/pimple/pimple/lib/Pimple.php:83
PHP  10. Bolt\Provider\ConfigServiceProvider->Bolt\Provider\{closure}() /Users/aspinelli/Projects/bolt/vendor/pimple/pimple/lib/Pimple.php:126
PHP  11. Bolt\Config->__construct() /Users/aspinelli/Projects/bolt/src/Provider/ConfigServiceProvider.php:15
PHP  12. Bolt\Config->getConfig() /Users/aspinelli/Projects/bolt/src/Config.php:43
PHP  13. Bolt\Helpers\String::makeSafe() /Users/aspinelli/Projects/bolt/src/Config.php:239
```

**Comment by GawainLynch** [comment]
`release/2.0` will throw that as you need to do `composer update` to change the packages.

The `master` branch is in the middle of a massive refactor right now, so some things might be broken :grin:

**Comment by tonicospinelli** [comment]
ok man, after `composer update` for 2.0, now it works :wink: 

so, about current bug, I made the fix and a little improve to show local extensions when this command is called as you can see at #3598

**Comment by GawainLynch** [comment]
Yep, I am just trying to finish my first coffee, a backlog of emails, etc, then I will respond on that PR.

**Comment by GawainLynch** [comment]
`<insert smile here></insert smile here>`

**Comment by GawainLynch** [comment]
Fixed in #3598
